r/tests/testthat/helper-skip.R

```diff
@@ -69,12 +76,14 @@ skip_on_linux_devel <- function() {
 }
 
 skip_if_r_version <- function(r_version) {
+  if (force_tests) return()
   if (getRversion() <= r_version) {
     skip(paste("R version:", getRversion()))
   }
 }
 
 process_is_running <- function(x) {
+  if (force_tests) return(TRUE)
```

Comment:
A comment here about what is going on here would be nice, since it's slightly different from other places

Reply:
Yeah makes sense, as it caught me too :D